cuckoo/data/analyzer/windows/modules/auxiliary/human.py

```diff
@@ -151,14 +188,76 @@ def run(self):
         if "human.click_buttons" in self.options:
             self.do_click_buttons = int(self.options["human.click_buttons"])
 
+        if "human.change_windows" in self.options:
+            self.do_change_windows = int(self.options["human.change_windows"])
+
+        if self.do_change_windows:
+            # Pick some random procs and spawn them
+            spawn = [
+                "calc.exe", "notepad.exe", "iexplore.exe", "cmd.exe",
+                "explorer.exe", "wmplayer.exe"
+            ]
+            # Spawn 2 random procs
+            hwnd = USER32.GetForegroundWindow()
+            for i in xrange(2):
+                app = random.choice(spawn)
+                log.debug("[HUMAN.CHANGE_WINDOWS] Spawning %s", app)
+                SHELL32.ShellExecuteA(hwnd, "open", app, "", "", SW_SHOW)
+
         while self.do_run:
             if self.do_click_mouse:
                 click_mouse()
 
             if self.do_move_mouse:
                 move_mouse()
 
+            if self.do_change_windows:
+                change_windows()
+
             if self.do_click_buttons:
                 USER32.EnumWindows(EnumWindowsProc(foreach_window), 0)
 
             KERNEL32.Sleep(1000)
+
+    @staticmethod
+    def handle_32770(frameHandle):
+        """
+          Handle IE11's "View Downloads" window
+        """
+        log.debug("Setting %s as foreground window", frameHandle)
+        USER32.SetForegroundWindow(frameHandle)
+        # Press RIGHT + ENTER to run the file
+        log.debug("Sending RIGHT + ENTER...")
+        USER32.keybd_event(VK_RIGHT, 0x4D, KEYEVENTF_EXTENDEDKEY | 0, 0)
+        USER32.keybd_event(VK_RIGHT, 0x4D, KEYEVENTF_EXTENDEDKEY | KEYEVENTF_KEYUP, 0)
+        USER32.keybd_event(VK_RETURN, 0x1C, KEYEVENTF_EXTENDEDKEY | 0, 0)
+        USER32.keybd_event(VK_RETURN, 0x1C, KEYEVENTF_EXTENDEDKEY | KEYEVENTF_KEYUP, 0)
+
+    @staticmethod
+    def handle_ieframe(frameHandle):
```

Comment:
I think this one worked pretty well as well, but vaguely recall in some cases this one does trigger on other toolbars other than download notification toolbar. Not sure if there is much we can do about it though, since I looked into it a bit and seems in >IE8 when IE changed how these windows are generated things got a lot more generic and hard to find and click buttons...

For example, I think this one triggers on bars that pop up for plugins if those haven't been pre-configured in registry (ie Adobe PDF notification bar for defaults if they haven't been set yet, etc)